spec/factories/mentor_call_off_contracts.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :mentor_call_off_contract do
+    lead_provider { build(:lead_provider, cpd_lead_provider: build(:cpd_lead_provider)) }
+    cohort { Cohort.current || create(:cohort, :current) }
+
+    recruitment_target { 6000 }
+    payment_per_participant { 1000.0 }
+    version { "1.0" }
```

Comment:
maybe `0.0.1` instead to mimic reality?

Reply:
Currently statement and call_off_contract factories have `contract_version { "1.0" }`, wanted to keep it consistent